levels/c2.md

```diff
@@ -9,8 +9,7 @@ You are autonomous on your job and start to make others grow
 
 ### Hats & How We
 
-- Identifies useful new Hats
-- Gives some Hats back to the team
+- Takes up multiple Hats, be fully accountant for it, make them circulate if needed  
```

Comment:
Changed this due to recent and multiples feedback on levels C2 where it's not always a good things to pass around an Hat (e.g when we have limited number of holder), and it kind of not represent what we fully want